Makefile

```diff
@@ -134,6 +137,7 @@ ifdef HIPCONFIG
   DFLAGS    += -DO_HIP
   CXXFLAGS  += $(HIPCONFIG)
   GPUCXX    ?= hipcc
+  GPUFLAGS  += -Wall -ferror-limit=1
```

Comment:
What does this do?

Reply:
Turn on all warnings and stop compiling after one error. I wanted more warnings for debugging. I'll turn off the error limit